### PR TITLE
blockdev_stream_base_itself:Block stream based on itself

### PR DESCRIPTION
**Pull request by aliang123**
Block stream based on itself

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:1989793

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_stream_base_itself.q35: PASS (83.10 s)

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to review, thanks.

**Comment by ngu-niny** [comment]
1-Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_base_itself Finshed 2021-08-10 19:55:36 2021-08-10 19:57:56 PASS 140.033196

Acked-by: Nini Gu <ngu@redhat.com>

File: qemu/tests/blockdev_stream_base_itself.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+    def _remove_images(self):
+        for img in self._trash:
+            sp_admin.remove_volume(img)
+
+    def post_test(self):
+        try:
+            if self.clone_vm.is_alive():
+                self.clone_vm.destroy()
+            #self._remove_images()
```
Comment (zhencliu) [inline qemu/tests/blockdev_stream_base_itself.py:57]: _remove_images is defined but never used?

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_stream_base_itself.q35: PASS (80.00 s)

File: qemu/tests/blockdev_stream_base_itself.py (hunk 1 above)
Comment (aliang123) [reply qemu/tests/blockdev_stream_base_itself.py]: Have updated, thanks zhenchao.

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**